### ISSUE TITLE
e2e tests no longer running

### ISSUE THREAD

**Issue by jarmoza** [neurobagel/annotation_tool#369]
Something has changed that is now preventing our e2e tests from running. See [log](https://github.com/neurobagel/annotation_tool/actions/runs/4254552056/jobs/7401245352).

The issue arises during `nuxt build` resulting in the following error followed by exit.

```
node:internal/crypto/hash:71
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
```

Neither `npm run dev` nor `npm run build` produce this error locally.

**Comment by jarmoza** [comment]
@surchs, @rmanaem Any thoughts on why this is newly occurring?

**Comment by surchs** [comment]
are you running on node v16.x (LTS)?

**Comment by rmanaem** [comment]
@jarmoza I believe I had the same issue in query tool because of node 18, switched to node 16 and it was resolved

**Comment by rmanaem** [comment]
See [e2e-test.yaml](https://github.com/neurobagel/query-tool/blob/56bb1a4eef2cb2b6670dccb4ab1a8f236a25f0da/.github/workflows/e2e-test.yaml#L12-L15) in query tool

**Comment by jarmoza** [comment]
@rmanaem Looks like there is no node setup section in the annotation tool `e2e-test.yaml`. Likely fix is to specify the version.

**Comment by surchs** [comment]
@jarmoza, I think the section of the workflow that @rmanaem shared does contain the node version:

```yaml
      - name: Set up node env
        uses: actions/setup-node@v2.1.2
        with:
          node-version: 16
```
or do you mean something else? If you are running things locally, then you would probably need to select the v16 LTS version of node manually.

**Comment by jarmoza** [comment]
@surchs That's what I was referring to, yes. Just realized there is a typo in my original comment. `npm` is **not** producing this error locally.

**Comment by jarmoza** [comment]
@surchs @rmanaem Shared the query tool yaml file. The annotation tool yaml doesn't specify node version currently.

**Comment by surchs** [comment]
oh, my bad. I hadn't realized that @rmanaem's snippet was from a different repo. You are correct, I was wrong. 

With that in mind: makes sense to me that the e2e workflow is failing, not sure why it wasn't failing before.

**Comment by rmanaem** [comment]
Yea I shared the fix I used in the query tool
I think there is a relatively new change in [cypress github actions](https://github.com/cypress-io/github-action) that bumped the default node version to 18

**Comment by jarmoza** [comment]
Well, that explains that! I'll take care of the change.

**Comment by rmanaem** [comment]
I believe the annotation tool runs actions v4 but I reckon the change was across different versions

**Comment by jarmoza** [comment]
@rmanaem PR for this at https://github.com/neurobagel/annotation_tool/pull/370

**Comment by surchs** [comment]
@jarmoza: can we close this thing?

**Comment by jarmoza** [comment]
@surchs Yes. Apologies, I think I have not yet habituated this last step of the PR process. Closed by #370.
